app/components/edit/EditPatientHandout.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+interface PatientHandout {
+  id: string;
+  language: string;
+  link: string;
+  [key: string]: string;
+}
+
+const EditPatientHandout = ({ index, item, handleItemChange }: {
+  index: number;
+  item: patientHandout;
+  handleItemChange: (handoutIndex: number, handout: patientHandout) => null;
+}) => {
+  const handleFieldChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    handout: patientHandout,
+  ) => {
+    const { value } = e.target;
+    const { field } = e.target.dataset;
+    if (field) {
```

Comment:
When would this ever be false? It might be better to throw an exception in that case instead of silently do nothing.

Reply:
Without this check, the TS compiler yells at me that `Type 'undefined' cannot be used as an index type.ts(2538)` I tried various things to fix this but to no avail. Any suggestions?

Reply:
Great, I have added error throwing if field is falsy. Thanks!